README.md

```diff
@@ -1 +1,11 @@
-# grafana
\ No newline at end of file
+# grafana
+
+## Deployment
+
+```bash
+git push
+```
+
+Grafana state is persisted across restarts through the use of a fly.io volume mounted at `/var/lib/grafana`.
+
+Volume snapshots can be used to restore a previous version of the installation.
```

Comment:
Please add a link to the relevant Fly.io docs here.

I think this is the right page: https://fly.io/docs/volumes/snapshots/#restore-a-volume-from-a-snapshot

Reply:
```suggestion
Volume snapshots can be used to [restore a previous version](https://github.com/filecoin-project/lotus/issues/11589) of the installation.
```